addons/nachschub/configs/CfgVehicles_USA.hpp

```diff
@@ -9,10 +9,6 @@ class TB_supply_usa_ammo : WRAPPER_NAME(Box_IND_Wps_F)
     class TransportMagazines
     {
         MACRO_ADDMAGAZINE(rhs_mag_30Rnd_556x45_M855A1_PMAG_Tracer_Red,40);      // Standard GewehrMag
-        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_soft_pouch_coyote,2);       // MGMag5.56
-        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_mixed_soft_pouch_coyote,2); // MGMag5.56 Tracer
-        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m62_tracer,3);                   // MGMag7.62 AP Tracer
-        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m61_ap,3);                       // MGMag7.62 AP
```

Comment:
Sicher das hier alle raussollen. Zumindest ein paar Magazine wären schön

Reply:
Die Magazine werden aufgeteilt in eine LMG und MMG Kiste direkt im nächsten PR.
class TB_supply_usa_ammo_lmg : WRAPPER_NAME(Box_IND_Wps_F)
{
    PUBLIC_NAME_CAT("LMG-Munition",USA);

    class TransportMagazines
    {
        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_soft_pouch_coyote,5);       // MGMag5.56
        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_mixed_soft_pouch_coyote,5); // MGMag5.56 Tracer
        MACRO_ADDMAGAZINE(TB_mag_100Rnd_556x45_Mk318_tracer,5);                 // MGMag5.56 C-Mag Tracer
        MACRO_ADDMAGAZINE(TB_mag_100Rnd_556x45_Mk318_dim,5);                    // MGMag5.56 C-Mag DIM
    };
};

class TB_supply_usa_ammo_mmg : WRAPPER_NAME(Box_IND_Wps_F)
{
    PUBLIC_NAME_CAT("MMG-Munition",USA);

    class TransportMagazines
    {
        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m62_tracer,5);                   // MGMag7.62 AP Tracer
        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m61_ap,5);                       // MGMag7.62 AP
        MACRO_ADDMAGAZINE(TB_mag_100Rnd_338_LS_Tracer,5);                       // MGMag8.6 Tracer
        MACRO_ADDMAGAZINE(TB_mag_100Rnd_338_LS_DIM,5);                          // MGMag8.6 DIM
    };
};

Reply:
Wir verlieren damit keine Magazine, sondern haben tatsächlich für die benötigten Waffen mehr Munition. Es ist ganz ganz selten, dass in einem Team mehrere MGs vorhanden sind, die auch noch unterschiedliche MGs nutzen.
Dementsprechend sollte es auch nicht dazu kommen, dass doppelt Kisten geschleppt werden müssen.

Reply:
Du siehst, dass die Anzahl im nächsten PR auf 5 für alle sich steigert.